pkgs/top-level/haskell-packages.nix

```diff
@@ -1,6 +1,16 @@
 { pkgs, callPackage, stdenv, buildPlatform, targetPlatform }:
 
-rec {
+let # These are the GHC versions that support building with integer-simple.
```

Comment:
@peti something like this:

https://github.com/LumiGuide/nixpkgs/commit/ae72fef9398333cb2f46f6be66f723bd3fca7216